boms/cloud-oss-bom/RELEASING.md

```diff
@@ -54,8 +54,10 @@ site](https://g3doc.corp.google.com/company/teams/cloud-java/tools/developers/re
 Several docs in this and other repositories need to be updated once the 
 new release is available on Maven Central.
 
-* Send pull requests that change the version in these documents:
+* Edit these Wiki pages:
     * https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM
+    * https://github.com/googleapis/google-oauth-java-client/wiki/Setup-Instructions
```

Comment:
I didn't find `google-oauth-client` in the Libraries BOM https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/snapshot/artifact_details.html. Does the [Setup Instructions: Maven](https://github.com/googleapis/google-oauth-java-client/wiki/Setup-Instructions#maven) work for you?

Reply:
You're right. It isn't. I wonder if we used to have oauth-client in the BOM?